### DISCUSSION TITLE
How to launch VSCode Diff when Verify fails?

### DISCUSSION THREAD

**Discussion by X9VoiD** [VerifyTests/Verify#578] [General]
I've started using `Verify` just recently, saw this line
![image](https://user-images.githubusercontent.com/10338031/183473758-dd534581-8e3d-4a71-a7f7-665ad768d8fb.png)
and this line
![image](https://user-images.githubusercontent.com/10338031/183473663-7627bfe9-a111-41ce-898e-0d5a36e86ace.png)
So I expected it to just work with VSCode, however upon failure of tests it doesn't launch any diff within VSCode. Is there some step that I'm missing? Because it's not obvious in the docs for both Verify and DiffEngine if I have to do something extra to make this work.

**Comment by SimonCropp** [comment]
what is the full path to your VSCode install?

**Comment by X9VoiD** [comment]
`/home/oscar/.nix-profile/bin/code`

Though I recommend restructuring the project a bit to be able to optionally search in `PATH` for the desired executable. (just sniffed around the repo, and saw paths being hardcoded)

**Reply by SimonCropp** [reply]
> search in PATH for the desired executable

happy to accept a PR for that

**Reply by X9VoiD** [reply]
Just submitted a [PR](https://github.com/VerifyTests/DiffEngine/pull/286) to DiffEngine. Is planning on doing the CLI definitions on a separate PR but if you feel like we should do it on the same PR then please do tell. I hope it is to your satisfaction!

**Comment by aalmada** [comment]
Hi! I'm new to verify. I'm using vscode on Windows with the code hosted on WSL. Like the original post author, no diff tool appears. What do you suggest to use in this scenario and how to configure it?

**Reply by SimonCropp** [reply]
@aalmada i dont know. your scenario is not something i have done.
